.github/CODEOWNERS

```diff
@@ -1,7 +0,0 @@
-# The Origami Core team is the default owner of all Origami-supported repositories.
-# This file adds the owner as a reviewer for all pull requests made to this repository.
-# It references owners by source order - the last matching pattern takes the most precedence.
```

Comment:
What was the reason for removing `.github/CODEOWNERS` and `.github/ISSUE_TEMPLATE.md`? Maybe good to reinstate and set to a customer products team?

Reply:
It's not! Thanks I'll create an issue to add it